templates/stats/context.tpl

```diff
@@ -0,0 +1,189 @@
+									:aria-pressed="timelineInterval === 'day'"
+									aria-describedby="context-stats-graph-title"
+									:disabled="!isDailyIntervalEnabled"
+									@click="setTimelineInterval('day')"
+								>
+									{translate key="stats.daily"}
+								</pkp-button>
+								<pkp-button
+									:aria-pressed="timelineInterval === 'month'"
+									aria-describedby="context-stats-graph-title"
+									:disabled="!isMonthlyIntervalEnabled"
+									@click="setTimelineInterval('month')"
+								>
+									{translate key="stats.monthly"}
+								</pkp-button>
+							</div>
+						</div>
+					</div>
+					<table class="-screenReader" role="region" aria-live="polite">
+						<caption>{translate key="stats.views.timelineInterval"}</caption>
+						<thead>
+							<tr>
+								<th scope="col">{translate key="common.date"}</th>
+								<th scope="col">{translate key="stats.views"}</th>
+							</tr>
+						</thead>
+						<tbody>
+							<tr	v-for="segment in timeline" :key="segment.date">
+								<th scope="row">{{ segment.label }}</th>
+								<td>{{ segment.value }}</td>
+							</tr>
+						</tbody>
+					</table>
+					<line-chart :chart-data="chartData" aria-hidden="true"></line-chart>
+					<span v-if="isLoadingTimeline" class="pkpStats__loadingCover">
+						<spinner></spinner>
+					</span>
+				</div>
+				<div class="pkpStats__panel" role="region" aria-live="polite">
+					<pkp-header>
+						<h2>
+							{translate key="stats.views"}
+							<tooltip
+								tooltip="{translate key="stats.context.tooltip.text"}"
+								label="{translate key="stats.context.tooltip.label"}"
+							></tooltip>
+							<spinner v-if="isLoadingItems"></spinner>
+						</h2>
+						<template slot="actions">
+							<pkp-button
+								ref="downloadReportModalButton"
+								@click="$modal.show('downloadReport')"
+							>
+								{translate key="common.downloadReport"}
+							</pkp-button>
+							<modal
+								close-label="{translate key="common.close"}"
+								name="downloadReport"
+								title={translate key="common.download"}
+								@closed="setFocusToRef('downloadReportModalButton')"
+							>
+								<p>{translate key="stats.context.downloadReport.description"}</p>
+								<table class="pkpTable pkpStats__reportParams">
+									<tr class="pkpTable__row">
+										<th>{translate key="stats.dateRange"}</th>
+										<td>{{ getDateRangeDescription() }}</th>
+									</tr>
+								</table>
+								<action-panel class="pkpStats__reportAction">
+									<h2>{translate key="context.context"}</h2>
+									<p>
+										{translate key="stats.context.downloadReport.downloadContext.description"}
+									</p>
+									<template slot="actions">
+										<pkp-button
+											@click="downloadReport"
+										>
+											{translate key="stats.context.downloadReport.downloadContext"}
+										</pkp-button>
+									</template>
+								</action-panel>
+								<action-panel class="pkpStats__reportAction">
```

Comment:
I'm not seeing where this second action panel shows up in the UI. The same for the issue and article stats pages. I may be missing something as I don't have actual stats data to test with, but I wanted to flag it.